crates/forge/src/lib.rs

```diff
@@ -310,7 +312,14 @@ fn run_tests_from_file(
     )
     .context("Failed setting up runner.")?;
 
-    pretty_printing::print_running_tests(&tests.relative_path, tests.test_cases.len());
+    let tests_source = if tests.relative_path == "src/lib.cairo" {
+        tests.package_name.as_str()
```

Comment:
What do you think about adding a `package` string here?
It could be nice to have such an output
```
Running 1 test(s) from erc20 package
```

Reply:
Also, where are those tests coming from? I assume they are unit tests collected from the entire package. Am I right?

Reply:
Yup, these are the tests from the project tree

Reply:
> What do you think about adding a package string here?

sounds good